docs/man/git-lfs.1.ronn

```diff
@@ -51,6 +51,8 @@ commands and low level ("plumbing") commands.
     Show information about Git LFS files in the index and working tree.
 * git-lfs-migrate(1):
     Migrate history to or from git-lfs
+* git-lfs-prune(1):
```

Comment:
[git-for-windows.github.io-master (1).zip](https://github.com/git-lfs/git-lfs/files/1710335/git-for-windows.github.io-master.1.zip)
[](url)